pre-docs/inventory_version2.md

```diff
@@ -234,6 +241,33 @@ google_compute_instance.app.1:
   zone = us-west1-a
 ```
 
+#### `data-lookup` plugins
```

Comment:
I think we should jsut call these config-lookup plugins until we can use them outside config.